From 7caed15e2b9ee7a8ba50331d394aa33a5efc6e83 Mon Sep 17 00:00:00 2001
From: Tom Stellard <tstellar@redhat.com>
Date: Thu, 12 Oct 2017 20:50:06 -0700
Subject: [PATCH] CMakeLists: Set the correct default version

The default version is used if cmake cannot infer the version from the git tags.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cb2605b..9e5ea51 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,7 +42,7 @@ include ( utils )
 include ( GNUInstallDirs )
 
 ## Setup the package version.
-get_version ( "1.0.0" )
+get_version ( "1.0.6" )
 
 set ( BUILD_VERSION_MAJOR ${VERSION_MAJOR} )
 set ( BUILD_VERSION_MINOR ${VERSION_MINOR} )
-- 
2.17.0

